pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_user-roles-in-pantheon.adoc

```diff
@@ -0,0 +1,20 @@
+[id="user-roles-in-pantheon_{context}"]
+= User roles in {ProductShortName}
+
+In {ProductShortName}, each user is assigned a role that determines the actions the user can perform.
+
+There are two user roles defined in {ProductShortName} - `Author` and `Publisher`. The following permissions are associated with the roles:
+
+`Author`::
+
+* Use Git to import their modules files into {ProductShortName}.
+* Revise existing modules.
+* Preview and stage modules.
+* Search and sort any modules imported into {ProductShortName}.
+* Manage module metadata that is configurable in the user interface.
+ 
+`Publisher`::
+* All the permissions associated with the `Author` role.
+* Publish modules to the {PublishingPortal} and the {PublishingPortalSearh}. 
+* Unpublish modules from the {PublishingPortal} and the {PublishingPortalSearh}.
+* Create and manage `Product` and `Product Version`.
```

Comment:
Authors can also create product versions. (At least I was able to on staging when signed in with only author privileges.)

I wouldn't format **Product** or **Product Version** as code. Just stay with unformatted, lower case product and product version.

Reply:
done

Reply:
@stoobie If what you say is true, then you have likely found a bug. It's not intended that authors can manage products in Pv2.